app/models/manageiq/providers/ovirt/inventory/collector/target_collection.rb

```diff
@@ -150,6 +150,10 @@ def templates
     t
   end
 
+  def advertised_images
+    []
```

Comment:
So we do handle targeted refresh of storage domains, but the `manager.ovirt_services.advertised_images` call only works for all storagedomains.

A good follow-up would be to extract the `advertised_images` method into the collector, then adding a targeted advertised_images that looks at `references(:storagedomains)`